docs/workflow_tasks/move_granules.md

```diff
@@ -0,0 +1,57 @@
+---
+id: move_granules
+title: Move Granules
+hide_title: true
+---
+
+# Move Granules
+
+This task utilizes the Cumulus Message Adapter to interpret and construct incoming and outgoing messages.
+
+Links to the npm package, task input, output and configuration schema definitions and more can be found on the auto-generated [Cumulus Tasks](../tasks) page.
+
+## Summary
+
+This task utilizes the incoming config.input_granules and the task input list of s3 URIs along with the rest of the configuration objects to do the following for a list of files assigned to a single collection:
```

Comment:
Can we put config.input_granules, etc in code blocks: `config.input_granules`

Reply:
👍  e01520ad2